internal/views/screen.go

```diff
@@ -212,6 +212,7 @@ func (s *screen) doSearch() {
 
 		s.g.Update(func(g *ui.Gui) error {
 			v, _ := s.g.View("body")
+			v.SetCursor(0, int(n))
```

Comment:
Doing this breaks the searches on the other views.  I suggest doing: 

if s.body.stack.name() == "root" {
         v.SetCursor(0, int(n))
}

so the cursor only moves on the root page search.  When you search inside a partition the cursor stays at the top and the rows that are seen change so that the first search result is rendered in the first row.  I think it's ok to not do that on the root view only.

Reply:
I wasn't sure about this myself. Thanks for pointing it out.